### ISSUE TITLE
 Loss = nan when training

### ISSUE THREAD

**Issue by zhangyuxuan321** [fcdl94/FSS#9]
<img width="496" alt="image" src="https://user-images.githubusercontent.com/54260208/160806819-5019b619-fe32-4e72-963a-7ffabacb355b.png">
When I  trained on the coco dataset, I met this problem, and I don't  know how to solve this. I have tried to reduce the learning rate but this problem also happened, could you please give me some advice on it?

**Comment by fcdl94** [comment]
Hello!

We noted this problem, but this should not give any error since the batch is discarded when the loss is NaN.

If you found issues in training, please let me know.
Best.
Fabio
